ccd-definition/UserProfile.json

```diff
@@ -62,6 +62,55 @@
     "WorkBasketDefaultCaseType": "CARE_SUPERVISION_EPO",
     "WorkBasketDefaultState": "Open"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "UserIDAMId": "raghu@croydon.gov.uk",
+    "WorkBasketDefaultJurisdiction": "PUBLICLAW",
+    "WorkBasketDefaultCaseType": "CARE_SUPERVISION_EPO",
+    "WorkBasketDefaultState": "Open"
+  },
```

Comment:
Do we have those users locally? If not please remove - that file is not used in pipeline, it's confusing I know sorry!

Reply:
Removed :) I may have added this when we seen the user profile error in the pipeline before i knew it was casued by ccd